README.md

```diff
@@ -3,16 +3,17 @@ chi
 
 [![GoDoc Widget]][GoDoc] [![Travis Widget]][Travis]
 
-`chi` is a small, fast and expressive router / mux for Go HTTP services built with net/context.
+`chi` is a small, idiomatic and composable router for building Go 1.7+ HTTP services. `chi` utilizes
+the new `context` package introduced in Go 1.7 to handle signaling, cancelation and request-scoped
+values across a handler chain.
 
-Chi encourages writing services by composing small handlers and middlewares with many or few routes.
-Each middleware is like a layer of an onion connected through a consistent interface (http.Handler or
-chi.Handler) and a context.Context argument that flows down the layers during a request's lifecycle.
+Chi encourages writing services by composing small handlers and middlewares constructed together
+from multiple sub-routers that make up the complete service API.
 
 In order to get the most out of this pattern, chi's routing methods (Get, Post, Handle, Mount, etc.)
-support inline middlewares, middleware groups, and mounting (composing) any chi router to another -
-a bushel of onions. We've designed the Pressly API (150+ routes/handlers) exactly like this and its
-scaled very well.
+support inline middlewares, middleware groups, and mounting (composing) any chi router to another.
+We've designed the Pressly API (150+ routes/handlers) exactly like this for the goals of productivity,
+maintainability and expression.
 
 ![alt tag](https://imgry.pressly.com/x/fetch?url=deeporigins-deeporiginsllc.netdna-ssl.com/wp-content/uploads/sites/4/2015/09/Tai_Chi2.jpg&size=800x)
```

Comment:
@pkieltyka Instead of this chi photo, we should show a simple usage example or benchmarks.